### DISCUSSION TITLE
Feature Request - way to change Hue, Saturation, Lightness & Contrast of a layer

### DISCUSSION THREAD

**Discussion by YarlBoro** [guillaumechereau/goxel#356] [Ideas]
As in description, tool that allows manipulating listed characteristics. In the 2D editors this is essential & very powerful feature, it would be as useful in Goxel. These allow user to adjust very basic aspects of the image. In the context of game dev, it i.e. makes preparing color variants of objects/characters much, much easier.

**Comment by guillaumechereau** [comment]
Yes, this should be possible with a new 'filter'.  I'll have a look.

**Comment by guillaumechereau** [comment]
<img width="809" alt="Screenshot 2024-06-26 at 3 58 42 PM" src="https://github.com/guillaumechereau/goxel/assets/107679/c26a36f0-e3f5-4e20-b11d-c6d3423df4c0">

I added a new filter with Hue/Lightness/Saturation controls.

**Comment by YarlBoro** [comment]
I've somehow missed Your post earlier, Guillaume

I've spend some time with this new filter and it works perfect. Thanks for this feature, much appriciated!

If you will ever revise/push further this panel, please consider adding contrast.
